benchmarks/user/README.md

```diff
@@ -211,6 +211,51 @@ Do not forget to pin processes, especially on NUMA systems; below, we do so with
 ```
 numactl --cpubind=0 --membind=0 python benchmark.py ...
 ```
+
+## Benchmarking performance
+
+In order for a user to benchmark the performance of an operator calculating
+GFlops/s, Operational Intensity and Execution timings the `bench` mode should
+be used while also providing as an environment variable `DEVITO_LOGGING=DEBUG`.
+
+So the isotropic acoustic wave forward Operator in a `512**3` grid, space order
+12, and a simulation time of 100ms:
+
+```
+`DEVITO_LOGGING=DEBUG` python benchmark.py bench -P acoustic -d 512 512 512 -so 12 --tn 100
+```
+
+A `.json` file will be created in a folder named `results` except if otherwise specified
+with the `-r` option specifying the results directory.
+
+
+## Plotting a roofline
+
+In order for a user to plot the results obtained from the `bench` in the previous
```

Comment:
I've iterated through what you wrote to try improving it. Lemme know what you think

_To generate a [roofline model](https://people.eecs.berkeley.edu/~kubitron/courses/cs252-S12/handouts/papers/RooflineVyNoYellow.pdf) from the results obtained in \`bench\` mode, one can execute \`benchmark.py\` in \`plot\` mode. For example, the command

\`\`\`
 python benchmark.py plot -P acoustic -d 512 512 512 -so 12 --tn 100 -a --max-bw 12.8
  --flop-ceil 80 linpack
\`\`\`

will generate a roofline model for the results obtained from

\`\`\`
python benchmark.py bench -P acoustic -d 512 512 512 -so 12 --tn 100 -a
\`\`\`

The \`plot\` mode expects the same arguments used in \`bench\` mode plus two additional arguments to generate the roofline:

* `--max-bw FLOAT`
     DRAM bandwidth (GB/s)
 * `--flop-ceil <FLOAT TEXT>`
     CPU machine peak. A 2-tuple (float, str) is expected, representing the performance ceil (GFlops/s) and how the ceil was obtained (ideal peak, linpack, ...), respectively

In addition, points can be annotated with the runtime value, passing the `--point-runtime` argument.

To obtain the DRAM bandwidth of a system, we advise to use [STREAM](http://www.cs.virginia.edu/stream/ref.html)

To obtain the ideal CPU peak, one should instantiate this formula
```
#[cores] · #[avx units] · #[vector lanes] · #[FMA ports] · [ISA base frequency]
```
More details [in this paper](https://arxiv.org/pdf/1807.03032.pdf).